scope3_methodology/corporate/model.py

```diff
@@ -0,0 +1,82 @@
+""" Model for computing corporate emissions for an organization """
+
+import utils.utils as utils
+from defaults.corporate_defaults import CorporateDefaults
+
+
+class CorporateEmissions:
+    """Raw emissions information about an org and methodology of how to calculate emissions"""
+
+    def __init__(
+        self,
+        defaults: CorporateDefaults,
+        office_emissions_mt_co2e_per_employee_per_month: float | None = None,
+        datacenter_emissions_mt_co2e_per_employee_per_month: float | None = None,
+        travel_emissions_mt_co2e_per_employee_per_month: float | None = None,
+        commuting_emissions_mt_co2e_per_employee_per_month: float | None = None,
+        overhead_emissions_mt_co2e_per_employee_per_month: float | None = None,
+        corporate_emissions_mt_co2e_per_employee_per_month: float | None = None,
+        employees: int | None = None,
+    ) -> None:
+        self.employees = employees
+        self.corporate_emissions_mt_co2e_per_employee_per_month = (
+            corporate_emissions_mt_co2e_per_employee_per_month
+        )
+        self.office_emissions_mt_co2e_per_employee_per_month = (
+            office_emissions_mt_co2e_per_employee_per_month
+            if office_emissions_mt_co2e_per_employee_per_month
+            else defaults.office_emissions_mt_per_employee_per_month
+        )
+        self.datacenter_emissions_mt_co2e_per_employee_per_month = (
+            datacenter_emissions_mt_co2e_per_employee_per_month
+            if datacenter_emissions_mt_co2e_per_employee_per_month
+            else defaults.datacenter_emissions_mt_per_employee_per_month
+        )
+        self.travel_emissions_mt_co2e_per_employee_per_month = (
+            travel_emissions_mt_co2e_per_employee_per_month
+            if travel_emissions_mt_co2e_per_employee_per_month
+            else defaults.travel_emissions_mt_per_employee_per_month
+        )
+        self.commuting_emissions_mt_co2e_per_employee_per_month = (
+            commuting_emissions_mt_co2e_per_employee_per_month
+            if commuting_emissions_mt_co2e_per_employee_per_month
+            else defaults.commuting_emissions_mt_per_employee_per_month
+        )
+        self.overhead_emissions_mt_co2e_per_employee_per_month = (
+            overhead_emissions_mt_co2e_per_employee_per_month
+            if overhead_emissions_mt_co2e_per_employee_per_month
+            else defaults.overhead_emissions_mt_per_employee_per_month
+        )
+
+    def validate(self) -> None:
```

Comment:
So if we don't have this data does this mean we fall over? I think we need to update this to be able to fall back to an average?

Reply:
Right now, yes we require employees or to know mt per month or we can't compute. You wrote up https://github.com/scope3data/methodology/issues/11 and should add in a way to fallback.